packages/components/src/data-model/word-query/lexical-data/data-objects/word-as-lexeme-data.js

```diff
@@ -0,0 +1,24 @@
+import LexicalDataResult from '@comp/data-model/word-query/lexical-data/result/lexical-data-result.js'
+import { Lexeme, Lemma, Homonym, HomonymGroup } from 'alpheios-data-models'
+
+export default class WordAsLexemeData {
+  constructor ({ word, language } = {}) {
+    this._word = word
+    this._language = language
+  }
+
+  async retrieve () {
```

Comment:
since the retrieve call is always passed a lexical data object, maybe it should be here even though it's not used (and will be empty)?

Reply:
Will add